app/Http/Controllers/Dashboard.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Services\BoardService;
+use App\Services\ProfileService;
+use App\Services\SessionService;
+use Carbon\CarbonInterval;
+
+class Dashboard extends Controller
+{
+    private SessionService $sessionService;
+
+    private BoardService $boardService;
+
+    private ProfileService $profileService;
+
+    public function __construct()
+    {
+        $this->sessionService = new SessionService();
+        $this->boardService = new BoardService();
+        $this->profileService = new ProfileService();
+    }
+
+    //Method to show dashboard and send necessary information
+    public function dashboard()
+    {
+        $session = $this->sessionService->lastSession();
+
+        //Extract data for X and Y Axis
+        if (! empty($session)) {
```

Comment:
You need to handle the case when sessions are empty. This is why the lints are failing. If session is empty,  xAxis and yAxis won't be defined.